resolwe/flow/executors/__init__.py

```diff
@@ -97,9 +103,10 @@ def run(self, data_id, script, verbosity=1):
 
         self.data_id = data_id
 
+        data_dir = settings.FLOW_EXECUTOR['DATA_DIR']
         dir_mode = getattr(settings, 'FLOW_EXECUTOR', {}).get('DATA_DIR_MODE', 0o755)
```

Comment:
We'll have a lot of this. Maybe we should write a helper function, e.g.:
```
settings_value(**keys)
```
that you would call in this case as:
```
settings_value('FLOW_EXECUTOR', 'DATA_DIR')
```
It would return the value on the arbitrary dictionary depth, say:
```
settings_value('TEMPLATES', 'OPTIONS', 'context_processors')
```
@tjanez what do you think? You usually have good suggestions for naming such functions.